README.md

```diff
@@ -1,2 +1,35 @@
-# go-opensubtitles
-Go library for accessing the OpenSubtitles API
+# go-opensubtitles #
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/99a6484956d0492db384e518e684e82a)](https://www.codacy.com/manual/TheForgotten69/go-opensubtitles?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=TheForgotten69/go-opensubtitles&amp;utm_campaign=Badge_Grade)
+
+go-opensubtitles is a Go library for accessing the [OpenSubtitles API](https://www.opensubtitles.com/docs/api/html/index.htm).
+
+## Install
```

Comment:
Codacy found an issue: [[heading-style] Headings should use atx-closed](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)